components/ui/events/index.js

```diff
@@ -293,13 +295,14 @@ function Events({ props, loadMore = true }) {
       />
       <div
         onClick={scrollToTop}
-        className={`w-8 h-8 text-whiteCorp bg-primary flex justify-center items-center rounded-lg shadow-xl ${
+        className={`w-14 h-14 flex justify-center items-center bg-whiteCorp rounded-lg shadow-xl ${
           scrollButton
             ? "fixed z-10 bottom-[82px] md:bottom-[90px] lg:bottom-[90px] right-8 flex justify-end animate-appear"
             : "hidden"
         }`}
       >
-        <ArrowUp className="w-5 h-5" aria-hidden="true" />
+        <NextImage src={Imago} className="p-1" width="30" height="30" />
+        {/* <ArrowUp className="w-5 h-5" aria-hidden="true" /> */}
```

Comment:
Perquè has comentat arrowup i has posat el logo?

Reply:
Perque desapareix el logo de la pantalla i no hi ha cap element que mantingui la marca. Es una idea per mantenir el la marca quan desapareix el menú

Reply:
Pero creus que la gent entendrà que això serveix per anar a dalt? Prefereixes que no s'amagui el menu?

Reply:
Lo del menú está bé. Ens dona mes espai d'usabilitat. El que em pica una mica a la vista es que es queda molt blanca la pantalla. Trobo que la idea de tornar al inici, i que l'inici sigui el logo desaparegut, i que torni a sortir quan cliques al boto scrollToTop, és intuitiu i tanca una mica aquesta mancança de marca. Si no ho veus clar ho trec, pero lo del menú que desapareix es bona idea, no ho canviïs.

Reply:
Ok, ho veig be!